forem/forem
# Bump rpush-redis and remove hiredis dependency

## What type of PR is this? (check all applicable)


## Description

The new releases of `rpush-redis` and `modis` gems now don't include `hiredis` as a dependency.

## Related Tickets & Documents

N/A

## QA Instructions, Screenshots, Recordings

Test suite coverage should be able to validate we're covered here. You could also try to connect to a TLS-only Redis to ensure `hiredis` is no longer taking control over connections (and failing due to TLS not being supported by them).

### UI accessibility concerns?

None - Backend only

## Added tests?

- [x] No, and this is why: It's only a dependency bump and existing test coverage should be enough

## [Forem core team only] How will this change be communicated?


## [optional] Are there any post deployment tasks we need to perform?

Keeping an eye on Redis connections to ensure nothing breaks when `hiredis` is no longer used would be good

## [optional] What gif best describes this PR or how it makes you feel?

![safety](https://media.giphy.com/media/Xd1EhywNhOjq5EPn5t/giphy.gif)
